content/v2/openapi.yml

```
@@ -5262,6 +5464,39 @@ components:
             required:
               - name
               - ip
+    RegistrantChangeCreate:
+      description: Contact change attributes
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              domain_id:
+                type: string
+              contact_id:
+                type: string
+              extended_attributes:
+                $ref: '#/components/schemas/ExtendedAttributes'
+            example:
+              domain_id: "20333"
+              contact_id: "2"
+              extended_attributes: { "x-fi-registrant-idnumber":"1234" }
+    RegistrantChangeCheck:
+      description: Contact change attributes
+      required: true
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              domain_id:
+                type: string
```

We likely do not limit the input to be strictly a string so we could update this to:
```suggestion
                type:
                  - string
                  - integer
```